docs/gcc-Tips.rst

```diff
@@ -27,32 +27,9 @@ You can find more details on the experimentation below:
 
 -  `Kaggle Paris Meetup #12 Slides <https://drive.google.com/file/d/0B6qJBmoIxFe0ZHNCOXdoRWMxUm8/view>`__
 
-Some explanatory pictures:
-
-.. image:: ./_static/images/gcc-table.png
-   :align: center
-   :target: ./_static/images/gcc-table.png
-
-.. image:: ./_static/images/gcc-bars.png
-   :align: center
-   :target: ./_static/images/gcc-bars.png
-
-.. image:: ./_static/images/gcc-chart.png
-   :align: center
-   :target: ./_static/images/gcc-chart.png
-
-.. image:: ./_static/images/gcc-comparison-1.png
-   :align: center
-   :target: ./_static/images/gcc-comparison-1.png
+Here is an explanatory picture:
 
 .. image:: ./_static/images/gcc-comparison-2.png
    :align: center
    :target: ./_static/images/gcc-comparison-2.png
-
-.. image:: ./_static/images/gcc-meetup-1.png
-   :align: center
-   :target: ./_static/images/gcc-meetup-1.png
-
-.. image:: ./_static/images/gcc-meetup-2.png
-   :align: center
-   :target: ./_static/images/gcc-meetup-2.png
+   :alt: picture with a chart grouped by compiler set of options using O2 mtune=core2 as the baseline. All the other 3 options are faster, with O3 mtune=native being the fastest.
```

Comment:
```suggestion
   :alt: picture with a chart grouped by compiler set of options using O2 mtune=core2 as the baseline. All the other 3 options are faster, with O3 M tune equals native being the fastest.
```

I think it might help screen readers to break this phrase up, what do you think?

Reply:
I checked with a screen reader and only separating the 'M' works well, the equals to operator is being read aloud. Your suggestion is good to go if both occurences of mtune be changed to M tune.

Reply:
ah right, thank you! I just edited the suggestion to work for both uses of `mtune`.